### ISSUE TITLE
automate au/qld/statewide cache updates

### ISSUE THREAD

**Issue by andrewharvey** [openaddresses/openaddresses#4092]
The au/qld/statewide upstream data is updated weekly however to download the data you need to:

1. Make a `GET` request, see https://github.com/openaddresses/openaddresses/blob/master/sources/au/qld/README.statewide.md
2. Receive an email
3. Download a link from that email

I've been doing this manually on an ad-hoc basis, but I'd like to automate this. Would it be possible to set this up on OpenAddresses infrastructure. https://aws.amazon.com/getting-started/projects/setup-email-receiving-pipeline/ describes how you can use AWS to receive emails so we should be able to automate this.

Who manages the AWS permissions and is this something we can set up on the OA AWS account?

**Comment by migurski** [comment]
This could be really cool. I have the ability to make new SES rules and attach them to a Lambda invocation. Would you be interested in building this yourself?

**Comment by andrewharvey** [comment]
Yeah that was the plan, SES receipt rule to Lambda https://docs.aws.amazon.com/ses/latest/DeveloperGuide/receiving-email-getting-started-receipt-rule.html then a lambda function to download the file to the s3 cache.

I don't have any experience with Lambda or receiving with SES, but happy to give it a go when I get a chance. But if you're keen and know how to do it, please go ahead!

**Comment by andrewharvey** [comment]
This could also apply to au/vic/statewide, which similar to qld sends out an email with a link.

**Comment by andrewharvey** [comment]
This also applies to au/nsw/statewide, and it would still be nice to implement, since the current workflow requires me to manually download the data each week.
